cli/config/src/lib.rs

```diff
@@ -50,8 +66,8 @@ impl Config {
     }
 
     fn xdg_config_file() -> Result<PathBuf> {
-        let xdg_path = dirs::config_dir()
-            .ok_or_else(|| anyhow!("Cannot determine config directory"))?
+        let xdg_path = etcetera::base_strategy::choose_base_strategy()?
```

Comment:
Seems like we don't always have to use the full qualifier, some items are re-exported.

```suggestion
        let xdg_path = etcetera::choose_base_strategy()?
```